nomad/plan_apply.go

```diff
@@ -705,7 +705,7 @@ func evaluateNodePlan(snap *state.StateSnapshot, plan *structs.Plan, nodeID stri
 // The plan is only valid for disconnected nodes if it only contains
 // updates to mark allocations as unknown.
 func isValidForDisconnectedNode(plan *structs.Plan, nodeID string) bool {
-	if len(plan.NodeUpdate[nodeID]) != 0 || len(plan.NodePreemptions[nodeID]) != 0 {
+	if len(plan.NodePreemptions[nodeID]) != 0 {
```

Comment:
@tgross  Reading the comments for this NodePreemptions field, it seems like we should remove this check too.  Do you agree?

Reply:
Interestingly, that field is deprecated... if we're looking at it at all shouldn't we be looking at `AllocsPreempted` instead? But yeah, I agree it's fine to remove.